frontend/src/metabase/search/components/SearchResult.styled.jsx

```diff
@@ -1,16 +1,19 @@
 import styled from "styled-components";
 
 import { color, lighten } from "metabase/lib/colors";
-
+import { space } from "metabase/styled-components/theme";
 import Link from "metabase/components/Link";
 import Text from "metabase/components/type/Text";
-import { space } from "metabase/styled-components/theme";
+import LoadingSpinner from "metabase/components/LoadingSpinner";
 
 function getColorForIconWrapper(props) {
-  if (props.item.collection_position) {
-    return color("saturated-yellow");
-  }
-  return props.type === "collection" ? lighten("brand", 0.35) : color("brand");
+  return !props.active
```

Comment:
We were trying to avoid nested ternaries and be more verbose for better readability

Reply:
Fixed